test/JDBC/expected/TestStoredProcedures.out

```diff
@@ -239,3 +239,152 @@ DROP PROCEDURE sp_test14
 #storedproc#!#prep#!#sp_test19#!#uniqueidentifier|-|a|-|5b7c2e8d-6d90-411d-8e19-9a81067e6f6c|-|output
 #storedproc#!#prep#!#sp_test19#!#uniqueidentifier|-|a|-|5b7c2e8d-6d90-411d-8e19-9a81067e6f6c|-|inputoutput
 #DROP PROCEDURE sp_test19
+
+CREATE PROCEDURE sp_test20 (@a INT, @b INT OUTPUT) AS BEGIN Select @a as a, @b as b; SET @b=100; SET @a=1000; END;
```

Comment:
Please add RPC tests as well, Just take the same sql batch and write the prep-exec cases.